src/server/routes/core.ts

```diff
@@ -30,4 +30,12 @@ router.get('/healthcheck', (_, res: Response) => {
   res.status(200).send('200 OK');
 });
 
+router.use('/robots.txt', (_, res: Response) => {
+  res.type('text/plain');
+  res.send(`User-agent: *
+Disallow: /user/
+Disallow: /signin?*returnUrl=https://discussion.theguardian.com/comment-permalink/
```

Comment:
Could this just be `/signin?*` or are there some queries that we want to be crawled?

Reply:
I have a feeling that it's fine to crawl sign-in etc, but this was specifically set up to make sure specific comments weren't crawled for some reason